### ISSUE TITLE
SelectOneMenu is inaccessible

### ISSUE THREAD

**Issue by phk31097** [primefaces/primefaces#1951]
## 1) Environment
- PrimeFaces version: 6.0.8
- Does it work on the newest PrimeFaces version? No
- Application server + version: jboss eap 6.4
- Affected browsers: Internet Explorer 11
- Screenreader: NVDA 2016.3

## 2) Expected behavior
All labels of the select one menu should be read (as in other browsers like Google Chrome).
Meaning that a select box does not have a label according to the screenreader which is confusing for the user.

See [here ](http://forum.primefaces.org/viewtopic.php?f=3&t=48216)and [here](http://stackoverflow.com/questions/32227579/aria-activedescendant-causes-that-label-is-not-read-in-ie).

## 3) Actual behavior
Labels are not read. Instead, it says "list"

## 4) Steps to reproduce
Open [the showcase](http://www.primefaces.org/showcase/ui/input/oneMenu.xhtml) with IE 11 and a screenreader.
A simple oneMenu is enough to reproduce the bug.

## 5) Sample XHTML
From the showcase:

`   <p:outputLabel for="console" value="Basic:" />
        <p:selectOneMenu id="console" value="#{selectOneMenuView.console}" style="width:125px">
            <f:selectItem itemLabel="Select One" itemValue="" />
            <f:selectItem itemLabel="Xbox One" itemValue="Xbox One" />
            <f:selectItem itemLabel="PS4" itemValue="PS4" />
            <f:selectItem itemLabel="Wii U" itemValue="Wii U" />
        </p:selectOneMenu>`

## 6) Sample bean
Not needed.

**Comment by kukel** [comment]
Why do you think this is a PF issue and not an IE11 one? (Since it works in Chrome, and PF does not implement things specifically for a browser or screenreader)

**Comment by phk31097** [comment]
Thank you for replying.

It is possible that this is an issue of the Internet Explorer.
I know neither ARIA-standards nor the PrimeFaces-implementation well enough.

In the forum I found "IE 8-9-10-11 and non-legacy versions of others." in context of supported browsers, so I would expect PrimeFaces-components to work with those.
Since it does not work I was hoping to find help on the forum or Github.
A solution for browser-compatibility-problems should not only be in my interest, but also in the interest of PrimeFaces. Any support would be appreciated.

**Comment by cagataycivici** [comment]
Hi,

This seems like a bug in IE that is fixed on Edge.

The aria properties we use are ignored on IE.

https://developer.microsoft.com/en-us/microsoft-edge/platform/issues/8571350/

Regards,

Cagatay Civici
PrimeTek
